### PR TITLE
Fix CodeQL Failures

### PR DESCRIPTION
**Pull request by donfiguerres**
I'm trying to include DarkReader in a Hugo theme I'm working on when the CodeQL scan reported failures. I've created a pull request since it's a security vulnerability and the Security Policy says that I can create a pull request. All three failures are about URL sanitation.

CodeQL result: https://github.com/hugo-toha/toha/pull/595/checks?check_run_id=6810164189
Recommendation from CodeQL: https://codeql.github.com/codeql-query-help/javascript/js-incomplete-url-substring-sanitization/#:~:text=Sanitizing%20untrusted%20URLs%20is%20an,a%20set%20of%20allowed%20hosts.

### CODE REVIEW

**Comment by Gusted** [comment]
Even though CodeQL is theoretical correct in this case. This not a security sensitive codepath of Dark Reader so it really doesn't matter it cannot be exploited in anyway. Just to make CodeQL happy I will merge the PR.

**Review by Gusted — APPROVED** [review]

File: src/inject/dynamic-theme/style-manager.ts [hunk 1]
```diff
@@ -46,7 +46,7 @@ export function shouldManageStyle(element: Node) {
                 element.rel.toLowerCase().includes('stylesheet') &&
                 !element.disabled &&
                 (isFirefox ? !element.href.startsWith('moz-extension://') : true) &&
-                !element.href.startsWith('https://fonts.googleapis.com')
+                ((new URL(element.href)).hostname != 'fonts.googleapis.com')
```
Comment (Gusted) [inline src/inject/dynamic-theme/style-manager.ts:49]:
```suggestion
                ((new URL(element.href)).hostname !== 'fonts.googleapis.com')
```

**Comment by donfiguerres** [comment]
Thanks @Gusted ! I've committed your suggested changes.

**Comment by Gusted** [comment]
Thanks!

**Pull request merged**